pkg/cloud_provider/file/fake.go

```diff
@@ -156,7 +155,7 @@ func (manager *fakeServiceManager) ResizeInstance(ctx context.Context, obj *Serv
 	return instance, nil
 }
 
-func (manager *fakeServiceManager) CreateBackup(ctx context.Context, obj *ServiceInstance, backupName string, backupLocation string) (*filev1beta1.Backup, error) {
+func (manager *fakeServiceManager) CreateBackup(ctx context.Context, obj *ServiceInstance, backupName string, backupLocation string) (*filev1beta1multishare.Backup, error) {
```

Comment:
Let's not change the filev1beta1 to filev1beta1multishare for everything yet. We use the multishare api to test experimentatl or preview features.

Reply:
agreed, this is intentional